__tests__/any-of-labels.spec.ts

```diff
@@ -0,0 +1,106 @@
+import {Issue} from '../src/classes/issue';
+import {IssuesProcessor} from '../src/classes/issues-processor';
+import {IIssuesProcessorOptions} from '../src/interfaces/issues-processor-options';
+import {DefaultProcessorOptions} from './constants/default-processor-options';
+import {generateIssue} from './functions/generate-issue';
+
+describe('any-of-labels option', () => {
+  test('should do nothing when not set', async () => {
+    const sut = new IssuesProcessorBuilder()
+      .emptyAnyOfLabels()
+      .issues([{labels: [{name: 'some-label'}]}])
+      .build();
+
+    await sut.processIssues();
+
+    expect(sut.staleIssues).toHaveLength(1);
+  });
+
+  test('should skip it when none of the issue labels match', async () => {
+    const sut = new IssuesProcessorBuilder()
+      .anyOfLabels('skip-this-issue,and-this-one')
+      .issues([{labels: [{name: 'some-label'}, {name: 'some-other-label'}]}])
+      .build();
+
+    await sut.processIssues();
+
+    expect(sut.staleIssues).toHaveLength(0);
+  });
+
+  test('should skip it when the issue has no labels', async () => {
+    const sut = new IssuesProcessorBuilder()
+      .anyOfLabels('skip-this-issue,and-this-one')
+      .issues([{labels: []}])
+      .build();
+
+    await sut.processIssues();
+
+    expect(sut.staleIssues).toHaveLength(0);
+  });
+
+  test('should process it when one of the issue labels match', async () => {
+    const sut = new IssuesProcessorBuilder()
+      .anyOfLabels('skip-this-issue,and-this-one')
+      .issues([{labels: [{name: 'some-label'}, {name: 'skip-this-issue'}]}])
+      .build();
+
+    await sut.processIssues();
+
+    expect(sut.staleIssues).toHaveLength(1);
+  });
+
+  test('should process it when all the issue labels match', async () => {
+    const sut = new IssuesProcessorBuilder()
+      .anyOfLabels('skip-this-issue,and-this-one')
+      .issues([{labels: [{name: 'and-this-one'}, {name: 'skip-this-issue'}]}])
+      .build();
+
+    await sut.processIssues();
+
+    expect(sut.staleIssues).toHaveLength(1);
+  });
+});
+
+class IssuesProcessorBuilder {
```

Comment:
I prefer to have all the "tests arrange" in the top of the file.
It makes more sense to me since it's the first thing to code and the first thing executed as well in the tests.
What do you think?

Reply:
Well, ideally that fixture would be moved into a separate file and enriched to be shared and reused by all tests. But even if it stays here, I don't really see much value having it on top in this case because 1) the test arrangement would be the instantiation of the builder, not its declaration, 2) the builder itself, albeit useful, is not that interesting, and anyone going through the test suite should be able to get what it does just by looking at how it's used tbh, and 3) having the `describe/test` blocks as close to the beginning of the file as possible is more practical, IMO.